motoman_driver/src/industrial_robot_client/joint_trajectory_action.cpp

```diff
@@ -62,7 +63,11 @@ JointTrajectoryAction::JointTrajectoryAction() :
   // A parameter named topics_list contains the respective information for each of the groups
 
   std::string value;
-  ros::param::search("topics_list", value);
+  if(!ros::param::search("topics_list", value))
+  {
+    ROS_ERROR_STREAM(industrial_robot_client::motoman_utils::TOPIC_LIST_ERROR_MSG);
+    throw;
```

Comment:
That is not the case.  It does work as expected, causing the node to crash with an unhandled exception.  For now, I'd like to accept this PR.  I have one in the works that pulls all this logic out and replaces it with a backwards compatible version.